docs/gatsby-node.js

```diff
@@ -24,4 +24,589 @@ exports.createPages = ({ actions }) => {
     toPath: '/docs/installation/installing-superset-using-docker-compose',
     isPermanent: true,
   });
-};
+  createRedirect({
+    fromPath: '/installation.html#getting-started',
```

Comment:
It looks like there's no redirect for plain ol' `/installation.html` (without #anchors) 

There are probably a lot of links in the wild to that, so it's worth testing that the direct page redirect works, and if not, adding the redirect into the setlist.

This probably applies to other pages too, but I haven't scoured the list.

Reply:
Hmm I see it on my end? I think its just missing in the Diff

Reply:
Looks like it's there now.